util.lua

```diff
@@ -255,4 +255,15 @@ function util.is_inside(child, parent)
   return util.is_inside(next_child, parent)
 end
 
+function util.oxford_join(list, sep, sep_last)
+  local count = #list
+  if count < 1 then
+    return ""
+  elseif count == 1 then
+    return tostring(list[1])
+  else
+    return table.concat({table.unpack(list, 1, #list - 1)}, sep) .. sep_last .. list[#list]
```

Comment:
I don't think you need the unpacking/rebuilding; `table.concat(list, sep, 1, #list-1)`?